server/prompts/celebrity_name_check.txt

```diff
@@ -0,0 +1,4 @@
+Is "{{name}}" commonly recongized as a celebrity, even a minor one, in any country?
```

Comment:
Not blocking, but we should have an email dataset for this prompt.

Reply:
as discussed, will do this later.